outputs.tf

```diff
@@ -10,32 +10,27 @@ output "fabric_connection_name" {
 
 output "fabric_connection_status" {
   description = "Connection provisioning status."
-  value       = module.equinix-fabric-connection.primary_connection.status
+  value       = tolist(module.equinix-fabric-connection.primary_connection.operation)[0].equinix_status
 }
 
 output "fabric_connection_provider_status" {
   description = "Connection provisioning provider status."
-  value       = module.equinix-fabric-connection.primary_connection.provider_status
+  value       = tolist(module.equinix-fabric-connection.primary_connection.operation)[0].provider_status
 }
 
 output "fabric_connection_speed" {
-  description = "Connection speed."
-  value       = module.equinix-fabric-connection.primary_connection.speed
```

Comment:
Speed is always in Mbps for Fabric connection resources, so removed the speed_unit output and updated the speed output description to note the unit